tests/test_list_projects.py

```diff
@@ -0,0 +1,137 @@
+#  Copyright (C) 2016 - Yevgen Muntyan
+#  Copyright (C) 2016 - Ignacio Casal Quinteiro
+#  Copyright (C) 2016 - Arnavion
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, see <http://www.gnu.org/licenses/>.
+
+projects = [
```

Comment:
This is increasing fragmentation, we should strive to have a single point of truth or this will lead to inconsistencies and make development harder. Can we get the list of projects from the library instead?

Reply:
Or probably we want to skip this test altogether

Reply:
I think we need to refactor the project with something like a factory pattern to get the projects instead of us importing them explicitly. I recommend we update this test with that change.

Reply:
I'd suggest to drop this test to avoid development friction until this is properly exposed from the library